Source/controls/devices/joystick.cpp

```diff
@@ -65,6 +70,20 @@ StaticVector<ControllerButtonEvent, 4> Joystick::ToControllerButtonEvents(const
 		case JOY_BUTTON_BACK:
 			return { ControllerButtonEvent { ControllerButton_BUTTON_BACK, up } };
 #endif
+#ifdef JOY_BUTTON_MENU
+		case JOY_BUTTON_MENU:
+			return { ControllerButtonEvent { ControllerButton_BUTTON_MENU, up } };
+#endif
+#ifdef JOY_BUTTON_VOLUME_UP
+		case JOY_BUTTON_VOLUME_UP:
+			if(up)increasevolume();

```

Comment:
Aren't the volume buttons used to adjust system volume by default like on other handhelds?
Anyway, there should be no event handling in here, joystick mapping should only translate joystick events to ControllerButtons.